internal/framework/data_source_kubectl_kustomize_documents.go

```
@@ -0,0 +1,193 @@
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"sigs.k8s.io/kustomize/api/filesys"
```

golangci-lint (2.12.2)

[error] SA1019: "sigs.k8s.io/kustomize/api/filesys" is deprecated: use sigs.k8s.io/kustomize/kyaml/filesys instead. 

(staticcheck)